scripts/exregional_run_fcst.sh

```diff
@@ -205,7 +205,9 @@ relative_or_null=""
 target="${FIXLAM}/${CRES}${DOT_OR_USCORE}mosaic.halo${NH3}.nc" # must use *mosaic.halo3.nc
 symlink="grid_spec.nc"
 if [ -f "${target}" ]; then
-  ln_vrfy -sf ${relative_or_null} $target $symlink
+  #ln_vrfy -sf ${relative_or_null} $target $symlink
+  echo "$target -> $symlink"
+  cp_vrfy --no-clobber $target $symlink
```

Comment:
Why do we need a copy instead of a link for these fix files?

Reply:
According to Dan, the author of these changes, he says:

I don't know which files are causing the high file system load.  I only know that it happens while reading files from the INPUT directory.

Reply:
What's the problem here? 

These files should not cause any problems to my understanding. "copy" files instead of "link" actually increase system load.

Reply:
Reply from Dan:

The load spike happens during ingestion by the forecast model application.  We have restriped the INPUT directory to spread the files contained therein across all available Lustre hardware.  That spreading only occurs if the files are copied to the INPUT directory.  A link in the INPUT directory that points to a file outside of the INPUT directory will probably have default striping.

We can certainly remove some of these copies and run a test to see if the spike returns.  If not, then there is no need to make a copy for that file.

Reply:
Annette, we will keep this PR open but only merge it when it is demonstrated that this is a must.

rrfs_a uses the same workflow. Do they need to make similar changes as in this PR?

Reply:
Dan tested the changes in ush/link_fix.sh and "found no significant impact on file system load."  So you can ignore the changes in link_fix.sh.  He is now testing the scripts/exregional_run_fcst.sh and found "There is definitely a bump up in file system load without the copies in scripts/exregional_run_fcst.sh.  I'll see if I can narrow in on which ones need to be copied."

I will keep you posted.

I don't know the answer to rrfs_a.  Let me see if I can find out.